collector.go

```diff
@@ -28,6 +28,9 @@ import (
 	"github.com/prometheus/snmp_exporter/config"
 )
 
+// 64-bit float mantissa: https://en.wikipedia.org/wiki/Double-precision_floating-point_format
+const float64Mantissa = 9007199254740992
```

Comment:
Careful, go constants are "special" and try and do all kinds of [magic](https://blog.golang.org/constants#TOC_10.). Better to use an explicitly typed variable here.

Reply:
Good point, thanks!